renku/cli/run.py

```diff
@@ -194,17 +194,22 @@
 program. For example, a program might accept a date range as input and access
 all files within that range during its execution.
 
-To address this issue, the program can dump a list of input and output files
-that it is accessing in ``inputs.txt`` and ``outputs.txt``. Each line in these
-files is expected to be the path to an input or output file within the
-project's directory. When the program is finished, Renku will look for
-existence of these two files and adds their content to the list of explicit
-inputs and outputs. Renku will then delete these two files.
+To address this issue, the program can dump a mapping of input and output files
+that it is accessing in ``inputs.yml`` and ``outputs.yml``. This YAML file
+should be of the format
+.. code-block:: YAML
+
+   - name: path
```

Comment:
Since the file should be a YAML dict we should remove the `-`. I also would add another entry to it:
```
name1: path1
name2: path2
```